src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -316,6 +415,7 @@ def _launch_instance(self, *, s3_files: list[tuple[str, str]] | None = None) ->
             InstanceType="t3.micro",
             IamInstanceProfile={"Name": self.instance_profile_name},
             SubnetId=self.subnet_id,
+            KeyName="windowstesting",
```

Comment:
Is a KeyPair required for running the windows tests? It wasn't previously for Linux. Or is this just some left over testing artifact?

Reply:
left over from testing.